calblueprint/berkeley-student-cooperative
# Danashi/updatedUserContext

# ✨ New in this PR ✨

## One liner: what did you do? 🛠
Added useEffect so that user persists through refresh and navigating through pages. User only resets/disappears when they click to signout. 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
updated auth.tsx to update user when authStateChanged and to sign the user in when they register

## How can the reviewer test your code? 👩‍💻
on "/" path, you can sign in and register and see the authUser get updated as you refresh 

## Any bugs you encountered or still having trouble with? 🐛
slight delay to update state on refresh 

## Resources 📔
_link online resources and related PRs_ 


🏚 cc: @gregoriiaaa
